ui/rogue/combat/sim.ts

```diff
@@ -64,7 +65,7 @@ const SPEC_CONFIG = registerSpecConfig(Spec.SpecCombatRogue, {
 			// Running just under spell cap is typically preferrable to being over.
 			const spellHitSoftCapConfig = StatCap.fromPseudoStat(PseudoStat.PseudoStatSpellHitPercent, {
 				breakpoints: [16.95, 16.96, 16.97, 16.98, 16.99, 17],
-				capType: StatCapType.TypeSoftCap,
+				capType: StatCapType.TypeThreshold,
```

Comment:
Threshold will always try to get the highest possible cap

Reply:
Not gonna lie, forgot I had switched this just to see what it did. Gonna put it back + have a few other changes in a bit.